### DISCUSSION TITLE
How to remove "h" and "m" from custom report in timespent fields?

### DISCUSSION THREAD

**Discussion by Lanetolsun** [shridhar-tl/jira-assistant#357] [Q&A]
I wrote a custom report, added a timespent and total timespent field. They appear as 2h, 1h 30m in excel export. I want them to appear as  2 , 1,5 so I can add a formula in Excel. 

I know I have to use expressions but couldn't find any example or  a reference. 

Thanks.

**Comment by shridhar-tl** [comment]
Hi @Lanetolsun, I hope you are referring to worklog report. Already worklog report have settings which can control this behavior. Please look at the formatting options in worklog report settings by clicking on the settings from top right corner.

**Comment by Lanetolsun** [comment]
Hello, 

It's a custom report and this field specifically, by default it returns as 1h 30m etc in report.  I just don't know what to write to see these hours as numbers. 1h30m -> 1,5   45m -> 0,75 etc. 
![resim](https://github.com/shridhar-tl/jira-assistant/assets/34169955/7ce6532e-cebe-417a-8a7f-e09f281ee917)

**Comment by Lanetolsun** [comment]
Is there still no way to do this simple request? I can't find it by trial and error.

**Comment by Lanetolsun** [comment]
Is there no one really using that "Edit expression" field? No examples in text FAQ, no show in videos. I can try basic things like .toString() but without ANY pointers it's impossible to find. 

What do i have to do to get a help from a developer?

**Comment by Lanetolsun** [comment]
Still no answer to this, still an issue.
